pkg/resourcemanager/azuresql/azuresqluser/azuresqluser.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	azuresql "github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/v3.0/sql"
 	_ "github.com/denisenkom/go-mssqldb"
+	"github.com/pkg/errors"
```

Comment:
without an amendment to the makefile (that builds old ASO), these can regress. I don't see that in this PR.

Reply:
The Taskfile targets don't run for these packages because they're not in `hack`

Reply:
I'll make a TODO for following up with changes to the Makefile to match.

Or do we want to migrate ASO to the same Task approach?